### ISSUE TITLE
Activity: Save numpy array as OME-TIFF

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#729]
Implementations:
- bioio
- ?

**Comment by felixS27** [comment]
Where should I put the script in, when I developed one? In the OME-Tiff module?
And can I push it to main or should I do a merge request?

**Comment by tischi** [comment]
Yes, in the OME-Tiff module, the question is whether there is already a suitable "Activity" for the script? If not we would need to think how to create one. This is maybe something that we should do together...

And also here: adding new stuff can be without a PR.

**Comment by tischi** [comment]
Hi, I don't understand... for the training material activities the scripts must be here in the training resources, don't they?
